app/src/main/java/com/epfl/drawyourpath/path/RunArrayAdapter.kt

```diff
@@ -0,0 +1,64 @@
+package com.epfl.drawyourpath.path
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ArrayAdapter
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.LayoutRes
+import com.bumptech.glide.Glide
+import com.epfl.drawyourpath.R
+import com.epfl.drawyourpath.utils.Utils
+import com.google.android.gms.maps.model.LatLng
+
+class RunArrayAdapter(context: Context, runs: MutableList<Run>, @LayoutRes val resource: Int = R.layout.past_run) : ArrayAdapter<Run>(context, resource, runs) {

```

Comment:
Maybe you can add a comment to explain in which situation this adaptation will be used. Because we have lots adapter relative to the runs like the RunAdaptatator